tgui/packages/tgui/interfaces/Vending.js

```diff
@@ -14,8 +14,7 @@ const VendingRow = (props, context) => {
     !data.onstation
     || product.price === 0
     || (
-      !product.premium
-      && !product.extended
+      !product.extended
```

Comment:
gonna note: `product.premium` was made up, both premium and hacked products set `extended = TRUE` (see [`ui_static_data`](https://github.com/yogstation13/Yogstation/blob/master/code/modules/vending/_vending.dm#L714))